### PR TITLE
Adding support for Centrifuge staking rewards

### PR DESCRIPTION
**Pull request by lucasvo**
This PR adds support for Centrifuge to the rewards collector. I also modified the script to allow customizing the output directory.

### CODE REVIEW

**Comment by jonasW3F** [comment]
will take a look soon, thanks for this

**Comment by jonasW3F** [comment]
If I run your version locally I get the following errors:

```
yarn run v1.22.17
$ node src/index.js
Error: ENOENT: no such file or directory, open 'out/Account 1 G1rrUNQSk7CjjEmLSGcpNu72tVtyzbWdUvgmSer9eBitXWf.json'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at exportVariable (file:///home/jonas/Documents/test/staking-rewards-collector/src/fileWorker.js:8:12)
    at main (file:///home/jonas/Documents/test/staking-rewards-collector/src/index.js:63:7) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'out/Account 1 G1rrUNQSk7CjjEmLSGcpNu72tVtyzbWdUvgmSer9eBitXWf.json'
}
Error: ENOENT: no such file or directory, open 'out/Account 1 G1rrUNQSk7CjjEmLSGcpNu72tVtyzbWdUvgmSer9eBitXWf.csv'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at writeCSV (file:///home/jonas/Documents/test/staking-rewards-collector/src/fileWorker.js:24:13)
    at main (file:///home/jonas/Documents/test/staking-rewards-collector/src/index.js:64:7) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'out/Account 1 G1rrUNQSk7CjjEmLSGcpNu72tVtyzbWdUvgmSer9eBitXWf.csv'
}
Error: ENOENT: no such file or directory, open 'out/Account 2 15fTw39Ju2jJiHeGe1fJ5DtgugUauy9tr2HZuiRNFwqnGQ1Q.json'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at exportVariable (file:///home/jonas/Documents/test/staking-rewards-collector/src/fileWorker.js:8:12)
    at main (file:///home/jonas/Documents/test/staking-rewards-collector/src/index.js:63:7) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'out/Account 2 15fTw39Ju2jJiHeGe1fJ5DtgugUauy9tr2HZuiRNFwqnGQ1Q.json'
}
Error: ENOENT: no such file or directory, open 'out/Account 2 15fTw39Ju2jJiHeGe1fJ5DtgugUauy9tr2HZuiRNFwqnGQ1Q.csv'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at writeCSV (file:///home/jonas/Documents/test/staking-rewards-collector/src/fileWorker.js:24:13)
    at main (file:///home/jonas/Documents/test/staking-rewards-collector/src/index.js:64:7) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'out/Account 2 15fTw39Ju2jJiHeGe1fJ5DtgugUauy9tr2HZuiRNFwqnGQ1Q.csv'
}
Error: ENOENT: no such file or directory, open 'out/Overview.csv'
    at Object.openSync (fs.js:462:3)
    at Object.writeFileSync (fs.js:1384:35)
    at writeOverviewCSV (file:///home/jonas/Documents/test/staking-rewards-collector/src/fileWorker.js:85:12)
    at main (file:///home/jonas/Documents/test/staking-rewards-collector/src/index.js:71:11) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'out/Overview.csv'
}
```

**Comment by lucasvo** [comment]
@jonasW3F yeah I did not add "out" to git but had it in the config - I added the folder, another way to fix this would be to just leave the prefix empty in the config. Let me if this solution is fine!

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The program takes several inputs in the `config/userInput.json` file.
 
 Staking Rewards:
 * **addresses**: A list of objects containing the `address` you want to parse the staking rewards, the `name` of your address and the `startBalance`. It also contains a field `network`, where users can specify their address belonging to: `Polkadot`, `Kusama`, `Moonriver`, `Moonbeam`, `Shiden`, `Astar`.
```
Comment (jonasW3F) [inline README.md:46]: centrifuge is missing here. Please also go through the whole Readme and add info CFG support to the respective places.

File: src/utils.js [hunk 2]
```diff
@@ -212,20 +212,27 @@ export function getTicker(network){
...
             ticker = 'ASTR';
             break;
+       case 'centrifuge':
+            ticker = 'CFG';
+            break;
     }
     return ticker;
 }
 
+export function getTokenName(network) {
```
Comment (jonasW3F) [inline src/utils.js:222]: Can you elaborate why we need his function?

Comment (lucasvo) [reply src/utils.js:222]: Trading of CFG started much much earlier with the erc20 wrapped version that is listed on coingecko as `wrapped-centrifuge` and thus the only way to get price data on CFG is by querying coingecko for WCFG and not for `centrifuge`.

File: src/utils.js [hunk 3]
```diff
@@ -273,6 +283,10 @@ export function checkPriceAvailablilty(userInput, network){
     if(end.valueOf() < 1642402800000 & network == 'astar' & priceData == 'true'){
         console.log('Your requested time window lies before prices are available for ' + network.toUpperCase() + '. Switching off price data.');
         priceData = 'false';
-    } 
+    }
+    if(end.valueOf() < 1626220800 & network == 'centrifuge' & priceData == 'true'){
```
[the remark this answers is not in the document]
Comment (lucasvo) [reply src/utils.js:287]: No exchange listed CFG (or WCFG) before the 14th of July. Anything before that time were exchanges with IOUs and is not actual trading of CFG.

[the remark this answers is not in the document]
Comment (jonasW3F) [reply src/utils.js:287]: Alright, as we only use staking rewards this istn't that crucial anyway

File: src/utils.js (hunk 2 above)
Comment (jonasW3F) [reply src/utils.js:222]: Alright

**Pull request merged**